cupyx/scipy/sparse/construct.py

```diff
@@ -4,7 +4,7 @@
 from cupyx.scipy.sparse import csc
 from cupyx.scipy.sparse import csr
 from cupyx.scipy.sparse import dia
-from cupyx.scipy.sparse.sputils import isscalarlike
+from cupyx.scipy.sparse.sputils import get_index_dtype, upcast, isscalarlike
```

Comment:
We follow [OpenStack style guideline](https://docs.openstack.org/hacking/latest/user/hacking.html).
Please avoid importing objects directly (even though previous code does it).

```suggestion
from cupyx.scipy.sparse import sputils
```